.github/workflows/linting-skipper.yml

```diff
@@ -0,0 +1,21 @@
+# If you change this name also do it in linting.yml
```

Comment:
Really cool to see how you handle skipping required workflows! We should apply this technique to many more cases :eyes:

Reply:
We will. 👀